scripts/check_chromium_revision.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2025 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import os
+import requests
+from time import time
+from google.cloud import storage
+
+DEFAULT_TIMEOUT = 600.0
+BUCKET_NAME = 'wpt-versions'
+NEW_REVISION_FILE = 'pinned_chromium_revision_NEW'
+OLD_REVISION_FILE = 'pinned_chromium_revision'
+PLATFORM_INFO = [
+    ("Win_x64", "chrome-win.zip"),
+    ("Win", "chrome-win.zip"),
+    ("Linux_x64", "chrome-linux.zip"),
+    ("Mac", "chrome-mac.zip")
+]
+SNAPSHOTS_PATH = "https://storage.googleapis.com/chromium-browser-snapshots/"
+PR_NUMBER = "50375"
+REPO_OWNER = "web-platform-tests"
+REPO_NAME = "wpt"
+
+
+def trigger_ci_tests() -> str | None:
+    # Reopen the PR to run the CI tests.
+    s = requests.Session()
+    s.headers.update({"Authorization": f"token {get_token()}"})
+    url = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{PR_NUMBER}"
```

Comment:
We should either:
1. Use the GitHub client python library
2. Manually [pin](https://docs.github.com/en/rest/about-the-rest-api/api-versions?apiVersion=2022-11-28) the `X-GitHub-Api-Version` header in each API call.